roles/backup/tasks/storage.yml

```diff
@@ -10,7 +10,7 @@
 - name: Set file copy command
   set_fact:
     copy_cmd: >-
-      cp -fr /var/lib/pulp/. {{ _backup_dir }}/pulp
+      cp -fr /var/lib/pulp/{artifact,media,scripts,tmp} {{ _backup_dir }}/pulp/
```

Comment:
do we really need to backup the tmp directory here ?

Reply:
@aknochow we can remove backup of `tmp` here.  We confirmed with the galaxy team on this point.  The contents of the tmp dir is mainly the PID for gunicorn, which will be automatically re-created.